### PR TITLE
Citus Cloud: Add documentation on logging

### PR DESCRIPTION
**Pull request by lfittl**
This adds documentation on how logging works on Citus Cloud, as well as instructions for adding Papertrail as a log destination:

screen shot 2016-09-02 at 6 17 41 pm

### CODE REVIEW

File: cloud/logging.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+Logging
+#######
+
+What Is Being Logged
```
Comment (begriffs) [inline cloud/logging.rst]: Might sound more natural as "What is Logged"

**Comment by begriffs** [comment]
Other than my tiny quibble this looks great.

**Comment by craigkerstiens** [comment]
:shipit:

**Comment by lfittl** [comment]
Thanks, merged!

**Pull request closed without merging**